settings.gradle

```diff
@@ -1,4 +1,2 @@
-rootProject.name='rxandroid-root'
```

Comment:
Keep this. Otherwise you'll get `rxandroid-rxandroid` as a module name for the nested project.